ansible_collections/arista/avd/molecule/eos_validate_state/inventory/host_vars/dc2-leaf3b.arista.com.yml

```diff
@@ -1,28 +1,29 @@
 ---
 # Custom interface structured_config with shutdown key missing to make sure the defaults are handled properly
```

Comment:
Do we still have some test in validate state using the old structured config format? Since validate state is reading structured config.

Reply:
Reverted the changes with comment for deprecated data-models